app/code/community/Zip/Payment/Model/Api/Charge.php

```diff
@@ -151,14 +151,15 @@ public function cancel($chargeId)
     public function prepareCreatePayload()
     {
         $chargeReq = new CreateChargeRequest();
-
+        $shouldCapture = $this->isImmediateCapture();
+        $capture = (strtoupper($this->getOrder()->getOrderCurrencyCode()) == 'AUD') ? $shouldCapture : true;
         $chargeReq
             ->setReference((string) $this->getOrder()->getIncrementId())
             ->setAmount((float) $this->getOrder()->getGrandTotal())
             ->setCurrency((string) $this->getOrder()->getOrderCurrencyCode())
             ->setOrder($this->getChargeOrder())
             ->setMetadata($this->getMetadata())
-            ->setCapture($this->isImmediateCapture())
+            ->setCapture($capture)
```

Comment:
Hi @mahbub-zip I think charge or not should be configured by merchant not hard coded by Region or Currency, Magento has auth and capture settings should use Mage Config settings